examples/hif2a/hifa2_config.yaml

```diff
@@ -0,0 +1,15 @@
+---
+map-generation:
+  strategy: geometry
+  protein: datasets/fep-benchmark/hif2a/5tbm_prepared.pdb
+  forcefield: ff/params/smirnoff_1_1_0_ccc.py
+  ligands:
+    - datasets/fep-benchmark/hif2a/ligands.sdf
+  # Cores and hubs must be in the same order. 
+  cores:
+    - "[*]~1~[*]~[*]~2~[*]~[*]~[*]~[*](~[#8]~[*]~3~[*]~[*]~[*]~[*]~[*]~3)~[*]~2~[*]~1"
+    - "[*]1~[*]~[*]~[*]~[*]~[*]1~[#8]~[*]2~[*]~[*]~[*]~[*]~[*]2"
+  hubs:
+    - "165"
```

Comment:
Definitely doable, and if we expand the manual selection we could probably easily allow users to modify the property in question they are specifying rather than using out convention of looking at ID

Reply:
I reworked it so that we can more easily add new methods (though it does risk the schema being too flexible). I believe this covers most use cases by partitioning the molecules by cores and running the different network generation methods (what were previously called hubs) on each set of cores.

Will likely want to write some more validation code to ensure the schema doesn't get out of control

Reply:
Nice! I think the current approach you've taken (dictionary of keyword arguments) is good for this stage of development, while we are adding several atom-mapping + edge-selection strategies.